File: Python_Qwasar_Platform/Exercises/my_isalpha.py
def my_isalpha(param_1):
	harf = 'qwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNM'
	for i in harf:
		if i == param_1:
			return 1
	else:
		return 0

print(my_isalpha("s"))

File: Python_Qwasar_Platform/Exercises/my_isalpha.c
#include <stdio.h>
#include <ctype.h>
    int my_isalpha(char param_1){
        if (param_1 >= 'A' && param_1 <= 'Z')
        return (1);
        if (param_1 >= 'a' && param_1 <= 'z')
        return (1);
    return (0);
}